lib/babe/babe_test.go

```diff
@@ -171,6 +177,133 @@ func TestMain(m *testing.M) {
 	os.Exit(code)
 }
 
+func newTestService_setupParameters(t *testing.T) (*Service, *state.EpochState, *types.BabeConfiguration) {
+	testDatadirPath, err := ioutil.TempDir("/tmp", "test-datadir-*")
+	require.NoError(t, err)
+
+	config := state.Config{
+		Path:     testDatadirPath,
+		LogLevel: log.LvlInfo,
+	}
+	dbSrv := state.NewService(config)
+	dbSrv.UseMemDB()
+
+	gen, genTrie, genHeader := newTestGenesisWithTrieAndHeader(t)
+	err = dbSrv.Initialise(gen, genHeader, genTrie)
+	require.NoError(t, err)
+
+	err = dbSrv.Start()
+	require.NoError(t, err)
+
+	t.Cleanup(func() {
+		_ = dbSrv.Stop()
+	})
+
+	rtCfg := &wasmer.Config{}
+	rtCfg.Storage, err = rtstorage.NewTrieState(genTrie)
+	require.NoError(t, err)
+	rt, err := wasmer.NewRuntimeFromGenesis(gen, rtCfg) //nolint
+	require.NoError(t, err)
+
+	genCfg, err := rt.BabeConfiguration()
+	require.NoError(t, err)
+
+	s := &Service{
+		epochState: dbSrv.Epoch,
+	}
+
+	return s, dbSrv.Epoch, genCfg
+}
+
+func TestService_setupParameters_genesis(t *testing.T) {
+	s, _, genCfg := newTestService_setupParameters(t)
```

Comment:
Don't use underscores in Go names
`newTestService_setupParameters` -> `newTestServiceSetupParameters`

Reply:
done